internal/postprocessor/README.md

```diff
@@ -13,41 +13,75 @@ fork instead of the repo.
 After following these steps the generated code will have replaced corresponding
 files in the `google-cloud-go` repo.
 
-## Docker container
+## Running the post-processor docker container locally
 
-The Docker container needs to be built with the context of the entire
-`google-cloud-go/internal` directory. When building the container, do so from
-the `google-cloud-go/internal` directory
+You can verify the name of the docker container name can be found in the
+`.github/OwlBot.yaml` and `github/OwlBot.lock.yaml` files.
 
-## Running the post-processor locally
-
-The Docker container name needed will be found in the `.github/OwlBot.yaml` and
-`github/OwlBot.lock.yaml` files.
-To run post-processor run:
+In the `google-cloud-go` root directory:
 
 ```bash
-docker pull <container-name>
-docker run --user $(id -u):$(id -g) --rm -v $(pwd):/repo -w /repo <container-name>
+docker pull gcr.io/cloud-devrel-public-resources/owlbot-go:latest
+docker run --user $(id -u):$(id -g) --rm -v $(pwd):/repo -w /repo gcr.io/cloud-devrel-public-resources/owlbot-go:latest
 ```
 
-## Testing the post-processor locally
+## Making changes, rebuilding the docker container and updating the OwlBot SHA
 
-You can run the post-processor locally on selected directories or on all of the
-clients in the root directory.
+After making changes to the post-processor, you need to publish a new version
+of the post-processor docker container and manually update the which version of
+the post-processor is used by OwlBot. To do this you need to update the SHA in
+the OwlBot lock file.
 
-### Run post-processor on all clients
+### Docker container
 
-From the `google-cloud-go/internal/postprocessor` directory run: 
-```bash
-go run main.go -client-root="../.." -googleapis-dir="/path/to/local/googleapis"
-```
+*Note*: The docker container needs to be built with the context of the entire
+`google-cloud-go/internal` directory. If building the container directly with
+`docker`, rather than with Cloud Build as instructed below, be sure to do so
```

Comment:
This is fine for now, but would you file an issue about making this easier. I don't think we should have to have the context of the internal directory anymore since I decoupled the postprocessor from gapicgen earlier this week.

Reply:
Can we just delete this *note*? I think building the container with Cloud Build is easy and simple.

Reply:
Deleted the note. Let me know if you want something added here, or if an issue needs to be filed.